web/Web/web/interfaces/SearchEngine/anserini.py

```diff
@@ -0,0 +1,31 @@
+import requests
+from config.settings.base import SEARCH_SERVER_IP
+from config.settings.base import SEARCH_SERVER_PORT
+from web.interfaces.SearchEngine.base import Search
+from collections import OrderedDict
+
+
+class Anserini(Search):
+
+    @staticmethod
+    def search(query: str, size: int):
+        response = requests.get(
+            f"http://{SEARCH_SERVER_IP}:{SEARCH_SERVER_PORT}/search",
+            params={
+                "query": query,
+                "size": size
+            }
+        )
+        response.raise_for_status()
+        results = response.json()
+        document_values = OrderedDict(results)
+        document_ids = document_values.keys()
+        return document_ids, document_values, response.elapsed.total_seconds()
+
+    @staticmethod
+    def get_content(docno: str) -> str:
+        raise NotImplementedError
```

Comment:
`get_content` shouldn't be hard to implement. For bigger collections that are usually compressed, nginx would be hassle to get it fetching document content.